doc/api/tls.md

```diff
@@ -620,6 +634,16 @@ Creates a new client connection to the given `port` and `host` (old API) or
     the list of supplied CAs. An `'error'` event is emitted if verification
     fails; `err.code` contains the OpenSSL error code. Default: `true`.
 
+  - `pskCallback(hint)`: When negotiating TLS-PSK, this optional function is
+    called with the identity hint provided by the server. If the client should
+    continue to negotiate PSK ciphers, the return value of this function must
+    be an object with string `identity`, and buffer `psk` properties.
+
+    Note that PSK ciphers are disabled by default, and using TLS-PSK thus
+    requires explicitely specifying a cipher suite with the `ciphers` option.
+    Additionally, you may need to disable `rejectUnauthorized` if you intend to
+    not use certificates.
```

Comment:
Overall the docs change is good but I would prefer not to use `you` in the docs. Perhaps:

```
Additionally, it may be necessary to disable `rejectUnauthorized` when not intending
to use certificates.
```